### PR TITLE
Am longlv new feature fragment argument

### PR DESCRIPTION
**Pull request by longlv81**

### CODE REVIEW

File: app/src/am_longlv/com/example/asian1/issue4fragmentargument/TwoFragmentsActivity.java [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+    private void createFragment(String method, Fragment fragment, int fragmentName, int color) {
+        Bundle bundle = new Bundle();
+        FragmentManager fragmentManager;
+
+        // Put color into Argument
+        bundle.putInt(SEND_COLOR, color);
+        fragment.setArguments(bundle);
+
+        // Create Fragment
+        fragmentManager = getSupportFragmentManager();
```
Comment (taitqs) [inline app/src/am_longlv/com/example/asian1/issue4fragmentargument/TwoFragmentsActivity.java:76]: FragmentManager fragmentManager = getSupportFragmentManager();

File: app/src/am_longlv/res/layout/activity_two_fragments.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (taitqs) [inline app/src/am_longlv/res/layout/activity_two_fragments.xml:2]: Use constrainlayout

File: app/src/am_longlv/res/layout/activity_two_fragments.xml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        android:layout_height="wrap_content">
+
+        <Button
+            android:id="@+id/btnFragmentOne"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_weight="1"
+            android:text="@string/fragment_one_button" />
+
+        <Button
```
[the remark this answers is not in the document]
Comment (taitqs) [reply app/src/am_longlv/res/layout/activity_two_fragments.xml:20]: Fix

**Review by huyhn30 — CHANGES_REQUESTED** [review]
wrong condition issue:
Have color in background first time open -> Don't have color until click button
Wrong title when backstack

**Review by huyhn30 — APPROVED** [review]

**Pull request merged**